website/pages/en/index.js

```diff
@@ -31,159 +40,413 @@ Button.propTypes = {
   children: React.PropTypes.any
 };
 
-class HomeSplash extends React.Component {
-  render() {
-    return (
-      <div className="homeContainer">
-        <div className="homeSplashFade">
-          <div className="wrapper homeWrapper">
-            <div className="projectLogo">
-              <img src="/icon.png" />
-            </div>
-            <div className="inner">
-              <h2 className="projectTitle">
-                {siteConfig.title}
-                <small>{siteConfig.tagline}</small>
-              </h2>
-              <div className="section promoSection">
-                <div className="promoRow">
-                  <div className="pluginRowBlock">
-                    <Button href="/playground/">Try It Out</Button>
-                    <Button
-                      href={"/docs/" + this.props.language + "/install.html"}
-                    >
-                      Get Started
-                    </Button>
-                    <Button
-                      href={"/docs/" + this.props.language + "/options.html"}
-                    >
-                      Options
-                    </Button>
-                  </div>
+const HomeSplash = props => {
+  return (
+    <div className="homeContainer">
+      <div className="homeSplashFade">
+        <div className="wrapper homeWrapper">
+          <div className="animatedLogoWrapper">
+            <AnimatedLogo />
+          </div>
+          <div className="inner">
+            <div className="section promoSection">
+              <div className="promoRow">
+                <div className="pluginRowBlock">
+                  <Button href="/playground/">Try It Out</Button>
+                  <Button href={"/docs/" + props.language + "/install.html"}>
+                    Get Started
+                  </Button>
+                  <Button href={"/docs/" + props.language + "/options.html"}>
+                    Options
+                  </Button>
                 </div>
               </div>
             </div>
           </div>
         </div>
       </div>
-    );
-  }
-}
+    </div>
+  );
+};
 
 HomeSplash.propTypes = {
   language: React.PropTypes.string
 };
 
-class Index extends React.Component {
-  render() {
-    const language = this.props.language || "en";
-    const showcase = siteConfig.users
-      .filter(user => {
-        return user.pinned;
-      })
-      .map((user, i) => {
-        return (
-          <a key={i} href={user.infoLink}>
-            <img src={user.image} title={user.caption} />
-            <br />
-            {user.caption}
-          </a>
-        );
-      });
+const TldrSection = ({ language }) => (
+  <div className="tldrSection productShowcaseSection lightBackground paddingTop paddingBottom">
+    <Container>
+      <div
+        style={{
+          display: "flex",
+          flexFlow: "row wrap",
+          justifyContent: "space-evenly"
```

Comment:
This doesn’t work on Safari. (Check out [MDN’s support table](https://developer.mozilla.org/en-US/docs/Web/CSS/justify-content#compat-desktop) for more details)

Reply:
If this had been written in CSS, not JS, it had been easy to add a fallback to `space-around`.

Reply:
I'll just change it to `space-around`.